apps/frontend/src/app/components/project/ProjectEdit.tsx

```diff
@@ -18,13 +25,36 @@ export const ProjectEdit = ({
     submit,
     loading,
 }: IProps): ReactElement => {
+    const [errorMessages, setErrorMessages] = useState<IInsightPojectErrors>(
+        {
+            nameErrorMsg: '',
+            descriptionErrorMsg: '',
+            datesErrorMsg: [],
+            projectTeamErrorMsg: ''
+        }
+    );
```

Comment:
A general comment: there will quickly become an abundance of snippets like this throughout the code, with small differences. I think validation might be a prime candidate for the Context API.